pypeit/reduce.py

```diff
@@ -1029,24 +1078,20 @@ def find_objects_pypeline(self, image, std_trace=None,
 
             sobjs.add_sobj(sobjs_slit)
 
-        # If this is a second pass for objfind or this is the first one but skip_second_find is True,
-        # assign slitmask design information to detected objects
+        # If this is a second pass for objfind or this is the first one but skip_second_find is True, extract some
+        # useful info from slitmask design. Matching to the detected objects is done in local_skysub_extract
```

Comment:
The comment on "Matching to the detected objects is done in local_skysub_extract" is no longer up to date. Also, does this need to be done here? Could/should it be done in PypeIt?